docker/rust_dockerfile

```diff
@@ -25,4 +25,4 @@ RUN cd / && \
     cd / && \
     rm -rf /rocksdb-4.12.fb /rocksdb.tar.gz
 
-RUN curl -sSf https://static.rust-lang.org/rustup.sh | sh -s  -- --date=2016-09-05 --disable-sudo -y --channel=nightly
+RUN curl -sSf https://static.rust-lang.org/rustup.sh | sh -s  -- --date=2016-09-29 --disable-sudo -y --channel=nightly
```

Comment:
I've upgraded rust to `2016-09-26` in jenkins-CI's image. This image just keeps pace with the version in CI